src/V3Undriven.cpp

```diff
@@ -381,7 +413,40 @@ class UndrivenVisitor final : public VNVisitor {
                     UINFO(9, " Full bus.  Entryp=" << cvtToHex(entryp) << endl);
                     warnAlwCombOrder(nodep);
                 }
-                entryp->drivenWhole();
+                if (entryp->isDrivenWhole() &&
+		    !m_inBBox &&
```

Comment:
Please run "make format". (github should have automatically, not sure why it didn't clean these up)